pyturbo/systems/compressor/compressor.py

```diff
@@ -81,23 +82,25 @@ def setup(self, init_file: Path = None):
         self.connect(self.aero.inwards, self.view.inwards, {"stage_count": "n"})
 
         # design methods
-        # scaling fan
-        scaling = self.add_design_method("scaling_fan")
-        scaling.extend(self.aero.design_methods["scaling"])
+        if config == "booster":
+            scaling = self.add_design_method("scaling")
+            scaling.extend(self.aero.design_methods["scaling_booster"])
```

Comment:
could be possible to do something like this ?  
```suggestion
            scaling.extend(self.aero.design_methods[f"scaling_{config}"])
```